colour/colorimetry/tests/test_tristimulus_values.py

```diff
@@ -13,6 +13,7 @@
 
 import numpy as np
 import unittest
+from pytest import approx
```

Comment:
**Major**

At this moment and because *pytest* is not a hard dependency, we should not depend on it and we should use `self.assertAlmostEqual` or `np.testing.assert_array_almost_equal`.

Reply:
Will pytest become a hard dev dependency in the future? It has a few nice features. And actually, I used some of them in a previous PR so you may want to check.

Reply:
Yes, I have considered it for a while but we will do that in one go, the `conftest.py` file in the `plotting` sub-package is fairly well isolated and more a convenient thing for people using `pytest`.

Reply:
Resolved in d98a69b8f444f426eec28ed43ba252f7e48a991f